2021-09-04 19:16:13.7128 INFO Now listening on: http://localhost:5013
2021-09-04 19:16:13.7814 INFO Application started. Press Ctrl+C to shut down.
2021-09-04 19:16:13.7991 INFO Hosting environment: Development
2021-09-04 19:16:13.8176 INFO Content root path: C:\Users\saeeduser\source\repos\AKC\AK.Api
2021-09-04 19:26:04.5172 INFO Now listening on: http://localhost:5013
2021-09-04 19:26:04.5897 INFO Application started. Press Ctrl+C to shut down.
2021-09-04 19:26:04.6051 INFO Hosting environment: Development
2021-09-04 19:26:04.6231 INFO Content root path: C:\Users\saeeduser\source\repos\AKC\AK.Api
2021-09-04 19:28:29.6073 INFO Now listening on: http://localhost:5013
2021-09-04 19:28:29.7127 INFO Application started. Press Ctrl+C to shut down.
2021-09-04 19:28:29.7827 INFO Hosting environment: Development
2021-09-04 19:28:29.8219 INFO Content root path: C:\Users\saeeduser\source\repos\AKC\AK.Api
2021-09-04 19:28:43.0974 ERROR An error occurred using the connection to database 'akcdatabase' on server 'tcp:akcserver.database.windows.net,1433'.
2021-09-04 19:28:43.2334 ERROR An exception occurred while iterating over the results of a query for context type 'AK.Infrastructure.Data.EFDbContext'.
Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot open database "akcdatabase" requested by the login. The login failed.
Login failed for user 'akcuser'.
   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:db50e097-12c7-4059-8907-1ebf53701010
Error Number:4060,State:2,Class:11
ClientConnectionId before routing:00cf1570-0d89-44c5-879f-ccd98f92e61e
Routing Destination:b6d2f22faccf.tr15635.westeurope1-a.worker.database.windows.net,11005
2021-09-04 19:28:43.5967 ERROR An unhandled exception has occurred while executing the request.
2021-09-04 19:29:43.5150 INFO Now listening on: http://localhost:5013
2021-09-04 19:29:43.5790 INFO Application started. Press Ctrl+C to shut down.
2021-09-04 19:29:43.6177 INFO Hosting environment: Development
2021-09-04 19:29:43.6735 INFO Content root path: C:\Users\saeeduser\source\repos\AKC\AK.Api
